### DISCUSSION TITLE
Filter out retweets in media view

### DISCUSSION THREAD

**Discussion by Fmoment1** [robertodoering/harpy#703] [Ideas]
I love that the media viewer includes retweeted media alongside directly posted media. It's very useful for me when I want to quickly go through my own timeline as I retweet a lot of art.

However, sometimes you want to go to a user's profile and see the posts that **only they have made**, and not someone else's post that they retweeted. 

To this end, I suggest adding an option to "filter out retweeted media" in the media viewer, something which we can toggle on and off in that menu. That way, we only see what has been directly posted.

**Comment by robertodoering** [comment]
Currently the media list is linked to the home / user timeline.
So if you don't want to see retweeted media, you could filter out retweets in the timeline filter.

In the future I plan to add a separate filter for the media timelines, but for now you can filter the timeline to have the same result.

**Reply by Fmoment1** [reply]
Whoa, I didn't think to try that. Thank you!
